src/audio/heap.c

```diff
@@ -770,6 +771,7 @@ void *get_bank_or_seq_inner(s32 poolIdx, s32 arg1, s32 bankId) {
             loadedPool = &gBankLoadedPool;
             break;
         case 2:
+        default:
```

Comment:
this as well?

Reply:
Yes, set loadedPool to NULL at declaration and remove the default case.

Reply:
done